.github/workflows/docker-image.yml

```diff
@@ -0,0 +1,61 @@
+name: Docker
+
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+on:
+  push:
+    branches: [ main, master ]
+    # Publish semver tags as releases.
+    tags: [ 'v*.*.*' ]
+  pull_request:
+    branches: [ main, master ]
```

Comment:
I don't think we need to run on PRs at all. Then we can remove the conditonal steps from below.

Reply:
will remove